README.md

```diff
@@ -20,6 +20,28 @@ This action for [Changesets](https://github.com/changesets/changesets) creates a
 - published - A boolean value to indicate whether a publishing has happened or not
 - publishedPackages - A JSON array to present the published packages. The format is `[{"name": "@xx/xx", "version": "1.2.0"}, {"name": "@xx/xy", "version": "0.8.9"}]`
 
+### Permissions
+
+If your repository has [restrictive default permissions](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), you need to set the following permissions:
+
+```yaml
+permissions:
+  contents: write
+  pull-requests: write
+```
+
+You also need to check `Allow GitHub Actions to create and approve pull requests` in your repositories Actions settings.
+
+### Triggering other workflows
+
+When using the built-in `GITHUB_TOKEN`, tags, releases and some pull request events created by this action won't trigger other workflows. From the [GitHub Docs docs](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow#triggering-a-workflow-from-a-workflow):
+
+> When you use the repository's `GITHUB_TOKEN` to perform tasks, events triggered by the `GITHUB_TOKEN`, will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs.
+
+To fix this, you should use a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) or a [GitHub App token](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/making-authenticated-api-requests-with-a-github-app-in-a-github-actions-workflow) for this action. You also need to set the `commitMode` input to `github-api`.
```

Comment:
We should also add a section of the `githup-api` mode and link to it from here. I think we should mention there it has some limitations (cc @s0), that it's creating signed commits and that it might increase the likelihood of running into rate limits (I have not verified that but it only makes sense to assume that).

Reply:
I think we could reduce the API requests when creating releases/tags in half, if you provide [`target_commitish`](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) to the [`createRelease` call](https://github.com/changesets/action/blob/main/src/run.ts#L44), and drop the [`createRef` call](https://github.com/changesets/action/blob/main/src/git.ts#L66).

> Specifies the commitish value that determines where the Git tag is created from. Can be any branch or commit SHA. Unused if the Git tag already exists. Default: the repository's default branch.

Reply:
I think it makes sense to highlight the impact on API usage on that commitMode definitely. 👍